### PR TITLE
feat: support customize image source in operator level

### LINKED ISSUE #701: Customize the repository and tag suffix in the operator
From v1.0.1 one operator supports a series of patch releases of the harbor, in this case, customize the special image of the component by the harbor version is not a good way to customize the images in the operator. 

Suggest customizing the default repository and tag suffix in the operator to replace the customize special image of the component with the harbor version.

### PR DESCRIPTION
**Pull request by heww**
Closes #701

BREAKING CHANGE: Customize the component image in the operator level is removed.

Signed-off-by: He Weiwei <hweiwei@vmware.com>

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -66,19 +66,7 @@ jobs:
...
-      REGISTRYCTL_DOCKER_IMAGE_2_3_0: ghcr.io/goharbor/harbor-registryctl:v2.3.0
-      REGISTRY_DOCKER_IMAGE_2_3_0: ghcr.io/goharbor/registry-photon:v2.3.0
-      CHARTMUSEUM_DOCKER_IMAGE_2_3_0: ghcr.io/goharbor/chartmuseum-photon:v2.3.0
-      NOTARYSERVER_DOCKER_IMAGE_2_3_0: ghcr.io/goharbor/notary-server-photon:v2.3.0
-      NOTARYSIGNER_DOCKER_IMAGE_2_3_0: ghcr.io/goharbor/notary-signer-photon:v2.3.0
-      TRIVY_DOCKER_IMAGE_2_3_0: ghcr.io/goharbor/trivy-adapter-photon:v2.3.0
-      HARBORCLUSTER_MINIO_DOCKER_IMAGE_2_3_0: ghcr.io/goharbor/minio:2021-04-06T23-11-00Z
-      HARBORCLUSTER_REDIS_DOCKER_IMAGE_2_3_0: ghcr.io/goharbor/redis:5.0-alpine
-      HARBORCLUSTER_POSTGRESQL_DOCKER_IMAGE_2_3_0: ghcr.io/goharbor/spilo-12:1.6-p3
+      IMAGE_SOURCE_REPOSITORY: ghcr.io/goharbor
```
Comment (steven-zou) [inline .github/workflows/tests.yml:69]: no tag suffix supported?

Comment (steven-zou) [reply .github/workflows/tests.yml:69]: default is empty tag suffix?

**Review by steven-zou — COMMENTED** [review]
comments

File: .github/workflows/tests.yml (hunk 1 above)
Comment (heww) [reply .github/workflows/tests.yml:69]: yes, there is no tag suffix for the images in the ghcr.io/goharbor, so the tag suffix env is not needed here.

This repository env here is used to improve the stability of the tests.

**Review by steven-zou — APPROVED** [review]
LGTM

**Review by cndoit18 — APPROVED** [review]
LGTM

**Pull request merged**